python3.8/cookiecutter-aws-sam-efs-python/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,107 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  {{ cookiecutter.project_name }}
+
+  Sample SAM Template for {{ cookiecutter.project_name }}
+
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
```

Comment:
It is fine to not do in this PR, but I feel that adding description to every resource with some explanation of why that resource is used in the context of this template will be helpful.